tests/autorestart/test_container_autorestart.py

```diff
@@ -21,6 +21,76 @@
+        "ERR swss#orchagent: :- removeLag: Failed to remove ref count 1 LAG PortChannel10008."
+        When teamd was restarted, there was an error message in the syslog: "ERR teamd#teamsyncd:
+        :- readData: netlink reports an error=-33 on reading a netlink socket."
+
+        Third, during pmon container was restarted due to ledd process was killed for testing,
+        xcvrd process would write an error message into syslog such as "ERR pmon#xcvrd[29]: :-initialize
+        GlobalConfig: Sonic database config global file doesn't exist at /var/run/redis/sonic-db/database_global.json."
+        thermalctld process would write an error message into syslog such as "ERR pmon#thermalctld[33]:
+        Caught exception while initializing thermal manager."
+
+        Fourth, if orchagent process was killed and swss container was restarted, then syncd process
+        would write error messages such as "ERR syncd#syncd: [none] driverEgressMemoryUpdate:1395
+        Error getting cosq for port 1.". At the same time, syncd process also wrote two WARNING messages
+        into syslog such as "WARNING syncd#syncd:- saiDiscover: skipping since it causes crash:
+        SAI_STP_ATTR_BRIDGE_ID". Since there was a keyword "crash" in these warning message, logAnalyzer
+        would fail.
+
+        Fifth, systemd would fire an error message:"ERR systemd[1]: Failed to start SNMP/TEAMD container." since
+        SNMP/TEAMD container hits the limitation of restart. route_check.py also wrote an error message into syslog.
+
+    """
+    monit_ignoreRegex = [
+        ".*ERR monit.*",
+    ]
+    swss_ignoreRegex = [
+        ".*ERR swss#orchagent.*removeLag.*",
+    ]
+    pmon_ignoreRegex = [
+        ".*ERR pmon#xcvrd.*initializeGlobalConfig.*",
+        ".*ERR pmon#thermalctld.*Caught exception while initializing thermal manager.*",
+    ]
+    syncd_ignoreRegex = [
+        ".*ERR syncd#syncd.*driverEgressMemoryUpdate.*",
+        ".*ERR syncd#syncd.*brcm_sai*",
+        ".*WARNING syncd#syncd.*saiDiscover: skipping since it causes crash.*",
+    ]
+    teamd_ignoreRegex = [
+        ".*ERR teamd#teamsyncd.*readData.*netlink reports an error=-33 on reading a netlink socket.*",
+    ]
+    systemd_ignoreRegex = [
+        ".*ERR systemd.*Failed to start .* container*",
+    ]
+    kernel_ignoreRegex = [
+        ".*ERR kernel.*PortChannel.*",
+    ]
+    other_ignoreRegex = [
+        ".*ERR route_check.*",
+    ]
+    if loganalyzer:
+        loganalyzer.ignore_regex.extend(monit_ignoreRegex)
+        loganalyzer.ignore_regex.extend(swss_ignoreRegex)
+        loganalyzer.ignore_regex.extend(pmon_ignoreRegex)
+        loganalyzer.ignore_regex.extend(syncd_ignoreRegex)
+        loganalyzer.ignore_regex.extend(teamd_ignoreRegex)
+        loganalyzer.ignore_regex.extend(systemd_ignoreRegex)
+        loganalyzer.ignore_regex.extend(kernel_ignoreRegex)
+        loganalyzer.ignore_regex.extend(other_ignoreRegex)
```

Comment:
@yozhao101 this is not exactly what I had in mind when asked for splitting. But I understand that you have to put all regex up because you are looping through all services and run test against them. you currently cannot really apply individual regex for individual service.

I'll approve this PR and let it go in. Please open a separate PR to parameterize the test so that the test will run for each  service individually, that way you can apply ignore regex per service.

Reply:
Sorry, I misunderstood what you ware meaning in our meeting. I will create a separate PR to improve the ignore regex.